packages/react/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@frctl/react",
+  "version": "0.0.0",
```

Comment:
Depends on what the package name will be, if it's a new one it can start from 0.0.0 but if we decide to keep it the same as the old one we should start with 0.2.1.

Reply:
Agree, I don't have a strong opinion on this, I'll let you decide.